src/commands/index.ts

```diff
@@ -0,0 +1,95 @@
+import * as vscode from 'vscode';
+import { getConfig } from '../utils/vscode-extensions';
```

Comment:
What's the benefit of that? I've never heard of it (this not implies it's not true what you say :wink:). Even `vscode` and `typescript` teams use the same syntax that I'm using here.